ansible/roles/girder-histomicstk/tasks/main.yml

```diff
@@ -1,61 +1,43 @@
 ---
 - name: NodeJS | Add PPA key
   command: "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68576280"
-  become: true
+  become: yes
+  become_user: root
 
 - name: NodeJS | Add PPA
   apt_repository:
-    repo: "deb https://deb.nodesource.com/node_6.x {{ ansible_distribution_release }} main"
-  become: true
+    repo: "deb https://deb.nodesource.com/node_4.x {{ ansible_distribution_release }} main"
+  become: yes
```

Comment:
Is there a reason to use `become: yes` over `become: true`?  Should we be consistent?

Similarly, the default value of `become_user` is `root`, so is there a reason to specify it explicitly?